implementations/rust/ockam/ockam_core/src/api.rs

```diff
@@ -516,7 +518,7 @@ impl<T: Encode<()>> ResponseBuilder<T> {
 }
 
 #[allow(unused_variables)]
-pub fn assert_request_match<'a>(schema: impl Into<Option<&'a str>>, cbor: &[u8]) {
+pub fn assert_request_match<'a>(struct_name: impl Into<Option<&'a str>>, cbor: &[u8]) {
```

Comment:
While we currently send structs we could also send enum bodies in requests. The previous naming reads better to me, as in "assert the request matches a schema".

Reply:
Previous naming was misleading, as there are 2 schemas in this function: what you're going to send, and schema.cddl. Not saying current naming is perfect